client/state/data-getters/index.js

```diff
@@ -157,6 +157,20 @@ export const requestGutenbergDraftPost = ( siteId, draftId ) =>
 		{ fromApi: () => data => [ [ draftId, data ] ] }
 	);
 
+export const requestGutenbergDemoContent = () =>
+	requestHttpData(
+		'gutenberg-demo-content',
+		http(
+			{
+				path: `/gutenberg/demo-content`,
+				method: 'GET',
+				apiNamespace: 'wpcom/v2',
+			},
+			{}
+		),
+		{ fromApi: () => data => [ [ 'gutenberg-demo-content', data ] ] }
+	);
```

Comment:
thanks for the change. this will now prevent us from sending a new request all the time